lib/packwerk/formatters/offenses_formatter_plain.rb

```diff
@@ -0,0 +1,23 @@
+# typed: strict
+# frozen_string_literal: true
+
+module Packwerk
+  module Formatters
+    class OffensesFormatterPlain < OffensesFormatter
```

Comment:
This could also just be called `Plain` and the other one just called `Default`. Let me know what you think is best.

Reply:
Yep great idea, moved.